enterprise/admin/views.py

```diff
@@ -493,12 +494,12 @@ def get_users_by_email(cls, emails):
 
         Returns:
             users: Queryset of users who exist in the OpenEdX platform and who were in the list of email addresses
-            missing_emails: List of unique emails which were in the original list, but do not yet exist as users
+            unregistered_emails: List of unique emails which were in the original list, but do not yet exist as users
         """
         users = User.objects.filter(email__in=emails)
         present_emails = users.values_list('email', flat=True)
-        missing_emails = list(set(emails) - set(present_emails))
-        return users, missing_emails
+        unregistered_emails = get_unregistered_emails(emails, present_emails)
```

Comment:
I think we should keep using sets. However we should convert the list we are passing set to a lowercase email list via list comprehensions.